exit/test_exit.c

```diff
@@ -0,0 +1,513 @@
+/*
+ * Phoenix-RTOS
+ *
+ *    POSIX.1-2017 standard library functions tests
+ *    HEADER:
+ *    - stdlib.h
+ *    - unistd.h
+ *    TESTED:
+ *    - exit()
+ *    - _exit() (_Exit equivalent)
+ *    - atexit()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h>
+#include <signal.h>
+#include <fcntl.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <errno.h>
+#include <unity_fixture.h>
+
+#define PATH1    "/tmp/testFile1"
+#define PATH2    "/tmp/testFile2"
+#define PATH3    "/tmp/testFile3"
+#define PATH4    "/tmp/testFile4"
+#define TEST_STR "test123"
+
+/* Set to random value */
+static int test_creatRes = 0x12345678;
```

Comment:
Just a suggestion, imho more descriptive when found in the code. 
```suggestion
static int sigHandleRes = 0x12345678;
```
Why should it be some random number? `-1` wouldn`t be enough?

Reply:
sigHandleRes indicates that the value is returned by handler when its not. If it would be `-1` then when creat() failed then the value is not changed.